cfme/tests/openstack/infrastructure/test_relationships.py

```diff
@@ -24,7 +23,7 @@ def test_assigned_roles(provider):
     navigate_to(provider, 'Details')
     try:
         res = provider.get_detail('Relationships', 'Deployment Roles')
-    except NoSuchElementException:
+    except NameError:
```

Comment:
Why is it changed to this exception? [This line](https://github.com/ManageIQ/integration_tests/blob/master/widgetastic_manageiq/__init__.py#L491) raises `NoSuchElementException` if entry not found.

Reply:
I can see it behaves differently
https://rhos-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/Director/view/CFME_RHOS_Integration/job/rhos9_cfme58_integration/26/TEST_REPORT/

Reply:
Oh, it's `SummaryTable`, not `SummaryForm`.